### DISCUSSION TITLE
GPT 3 or GPT 3.5?

### DISCUSSION THREAD

**Discussion by ithou** [ddiu8081/chatgpt-demo#112] [Q&A] [answered]
### Describe the bug

用最新版代码在vercel部署的，问哪个版本回答 GPT 3
<img width="738" alt="image" src="https://user-images.githubusercontent.com/39048771/222961870-377192d2-dfd5-44de-a8bb-892ce4cdda5b.png">

### Reproduction

everytime

### System Info

```Shell
Chrome
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ddiu8081/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/ddiu8081/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by h7ml** [comment, marked as answer]
gpt-3.5-turbo

**Comment by xdtxAM** [comment]
啥意思，那到底是gpt3）还是3.5 turbo是啥
> gpt-3.5-turbo
